_pages/dev/performance/above-the-fold.md

```diff
@@ -56,17 +56,17 @@ ProductDetailsPageTemplate: {
 
 ## CSS Configuration
 
-By default, when page slots are loaded on the page, there's no minimum height of page slots or components available. Page slots will initially all have no height, which brings them all in the view port. This eliminates the concept of deferred loading, which is based on content not being the viewport.
+By default, when page slots are loaded on the page, there is no minimum height available for the page slots or components. All page slots initially have no height, which brings them all into the viewport. This prevents the deferred loading technique from working, because it depends on content not being in the viewport.
```

Comment:
I like a little better explanation here, something like:

> By default, when page slots are loaded on the page, there is no minimum height available for the page slots or components. The actual height is only added when components are loaded, and the associated CSS rules are applied to the components. The page slot will adjust its height automatically when components are loaded. Therefor, page slots do not have an initial height, which is why they end up in the viewport initially. This prevents the deferred loading technique from working, because it depends on content not being in the viewport.

Reply:
Very nice!
Done.